src/main/common/streambuf.c

```diff
@@ -148,7 +148,7 @@ bool sbufReadI8Safe(int8_t *dst, sbuf_t *src)
 
 bool sbufReadU16Safe(uint16_t *dst, sbuf_t *src)
 {
-    if (sbufBytesRemaining(src) >= sizeof(uint16_t)) {
+    if (sbufBytesRemaining(src) >= 2) {
```

Comment:
why change to a magic number?

Reply:
Because `sizeof` is unsigned and also this function reads 2 bytes, not `sizeof(uint16_t)` bytes (although they happen to be the same).

Reply:
Good point - check for number of bytes we need. Please add a comment explaining this.

Reply:
Is a comment really necessary? We are reading a U16, so we need to check if there are at least 2 bytes left in the buffer.

Reply:
Well, I didn't understand that without your explanation. Probably other developer might not figure that out as well.